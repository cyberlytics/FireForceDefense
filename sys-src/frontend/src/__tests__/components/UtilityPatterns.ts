import { mount } from '@vue/test-utils';
import UtilityPatterns from '@components/utilityPatterns.vue';

describe('UtilityPatterns.vue', () => {
    test('utility patterns match snapshot', () => {
        const wrapper = mount(UtilityPatterns);
        expect(wrapper.element).toMatchSnapshot();
    });
});
