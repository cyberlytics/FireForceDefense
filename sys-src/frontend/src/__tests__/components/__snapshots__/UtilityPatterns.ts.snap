// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UtilityPatterns.vue utility patterns match snapshot 1`] = `
<defs>
  <pattern
    height="1"
    id="utility-Coin"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/utilities-Coin.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="utility-Abbauen"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/utilities-Abbauen.svg"
      y="0"
    />
  </pattern>
  <pattern
    height="1"
    id="utility-Nothilfe"
    preserveAspectRatio="xMidYMid meet"
    viewBox="0 0 100 100"
    width="1"
  >
    <image
      height="100"
      width="100"
      x="0"
      xlink:href="/assets/utilities-Nothilfe.svg"
      y="0"
    />
  </pattern>
</defs>
`;
